src/components/InputSearch/InputSearch.tsx

```diff
@@ -61,17 +76,18 @@ export default function InputSearch(props: IInputProps) {
     labelClass.push(scss.labelError);
   };
 
-  if (
-    isFocused ||
-    (typeof props.value === 'string' && props.value.length > 0)
-  ) {
+  if (isFocused || (typeof value === 'string' && value)) {
```

Comment:
```suggestion
  if (isFocused || (typeof value === 'string')) {
```
btw pq checar se é string se value já tem essa tipagem?

Reply:
Meu aconteceu um erro que subi erradamente algumas coisas no pr, mas corrigirei no lugar certo.